README.md

```diff
@@ -56,6 +56,10 @@ There are two main categories of samples: **Camera** and **Applications**. The s
     - [**MultiCamera**][MultiCamera-url] ([tutorial][MultiCameraTutorial-url])
       - [**MultiCameraCalibration**][MultiCameraCalibration-url] and [**MultiCameraCalibrationFromFile**][MultiCameraCalibrationFromZDF-url] - Use captures of a calibration object to generate transformation matrices to a single coordinate frame. First sample captures from connected cameras, the second loads existing captures from .ZDF files.
       - [**StitchByTransformation**][StitchByTransformation-url] and [**StitchByTransformationFromFile**][StitchByTransformationFromZDF-url] - Use transformation matrices from Multi-Camera calibration to transform point clouds into single coordinate frame. First sample uses connected cameras, the second loads existing captures from .ZDF files.
+    - [**TransformViaArucoMarker**][TransformViaArucoMarker-url] - Transform a point cloud from camera frame to an aruco marker frame using the marker's pose.
```

Comment:
Transform a point cloud from **camera frame** to **an aruco marker frame** using the marker's pose

On aruco you have indefinite article **an** - get rid of it or add an article to camera frame as well. Compact solution is '..from camera to aruco frame using marker's pose' (which is what I would chose)

Reply:
agree, done

Reply:
It is better now. nitpick - we don't use 'a' before 'point cloud' in other sample descriptions, we agreed adding all articles would be verbose.